.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
Please remove the vscode files and do not sync them with GitHub. They should be added to the .gitignore file

Reply:
Sorry, I removed the file. It was a testing use. I forgot to delete it after I used it.